docs/es/docs/tutorial/path-params.md

```diff
@@ -0,0 +1,244 @@
+# Parámetros de path
+
+Puedes declarar los "parámetros" o "variables" con la misma sintaxis que usan los format strings de Python:
+
+```Python hl_lines="6-7"
+{!../../../docs_src/path_params/tutorial001.py!}
+```
+
+El valor del parámetro de path `item_id` será pasado a tu función como el argumento `item_id`.
+
+Entonces, si corres este ejemplo y vas a <a href="http://127.0.0.1:8000/items/foo" class="external-link" target="_blank">http://127.0.0.1:8000/items/foo</a>, verás una respuesta de:
+
+```JSON
+{"item_id":"foo"}
+```
+
+## Parámetros de path con tipos
```

Comment:
Talvez sea interesante poner `Tips` que usamos la palabra "path" en este issue: https://github.com/tiangolo/fastapi/issues/1218

Reply:
"path" is explained here: https://fastapi.tiangolo.com/es/tutorial/first-steps/#paso-3-crea-un-operacion-de-path so I think it's fine to not add more.

Also, adding a Tip here that is not in the original English version would start diverging from the original content, and it would be difficult to decide when to diverge and add more things and when not, so for now I'll leave it as is.